mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/VeilederflateService.kt

```diff
@@ -391,9 +400,12 @@ class VeilederflateService(
             }
     }
 
-    suspend fun hentOppskrifterForTiltakstype(tiltakstypeId: String, perspective: SanityPerspective): List<Oppskrift> {
+    suspend fun hentOppskrifterForTiltakstype(
+        tiltakstypeId: String,
```

Comment:
En grunn til at denne er en string og ikke en UUID? Er det fordi funksjonen er ment å støtte `draft.$uuid`?

Reply:
Er vel jeg som skrev den kodesnuttet, men kan ikke huske om det var noen spesiell grunn til at jeg gikk for String og ikke UUID. Du må gjerne prøve å endre til UUID.